### ISSUE TITLE
Filling of big relations confusing for new users

### ISSUE THREAD

**Issue by Mannivu** [openstreetmap/iD#9222]
### Description

In the screenshot there's a straight blue line that new users can find hard to understand what it represents. I understand the point of having it, but I think that this kind of filling could be scrapped for multipolygons that are so big they can't be fully displayed.

### Screenshots

![image](https://user-images.githubusercontent.com/42747216/181576839-ada4b3d6-1596-4bf6-9d49-98caffd104f6.png)

**Comment by pnorman** [comment]
Related: #1201

**Comment by tyrasd** [comment]
Yes, it's essentially the same issue as #1201. This particular "glitch" could be avoidable by special handling the "closing" segment when the multipolygon relation is not fully loaded. But let's track this in the other issue as well.
